### PR TITLE
Update README and the action permissions

### PR DESCRIPTION
**Pull request by mertcelebi**

### CODE REVIEW

File: .github/workflows/setup.yml [hunk 1]
```diff
@@ -8,6 +8,10 @@ jobs:
   setup-repo:
     if: ${{ github.repository != 'Arcadia-Science/nextflow-template-repository' }}
     runs-on: ubuntu-latest
+
+    permissions:
+      actions: write
```
Comment (mertcelebi) [inline .github/workflows/setup.yml]: This is a test to see if I can fix this without the permissions settings FYI

File: README.md [hunk 2]
```diff
@@ -8,12 +8,20 @@ This is fully inspired by the following blog post: https://simonwillison.net/202
 
 ## Usage
 
-You can start [here](https://github.com/Arcadia-Science/nextflow-template-repository/generate) and follow the prompts. Alternatively, you can click on the "Use this template" button on this repository and follow the prompts.
+You can click on the "Use this template" button on this repository and follow the prompts. Alternatively, you can start [here](https://github.com/Arcadia-Science/nextflow-template-repository/generate) and follow the prompts. Please follow these instructions:
+
+- The name of your repository will be the name of the Nextflow pipeline.
+- Add a one-line description of your pipeline.
+- Make sure to check "Include all branches". This will copy the `TEMPLATE` branch that's used by `nf-core` to keep modules up-to-date and make following the `nf-core` guidelines easier.
```
Comment (theosanderson) [inline README.md]: To explain my confusion on this point -- this repo doesn't seem to have a `TEMPLATE` branch afaict

Comment (mertcelebi) [reply README.md]: Yup, I am aware, that'll be added once this PR is merged. That was an oversight on our end

**Review by taylorreiter — APPROVED** [review]
lgtm!

**Review by elizabethmcd — APPROVED** [review]
👍🏻

**Review by taylorreiter — APPROVED** [review]
LGTM :) i didn't run it so no promises it's perfect but looks reasonable!

**Pull request merged**